fn/templates/fnlb-deployment.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}-fnlb-depl
+  labels:
+    app: {{ template "fullname" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+spec:
+  replicas: 1
```

Comment:
Only one comment, which needn't get in the way of merging: do we want to make the initial size of this deployment specifiable via the helm parameters?